src/components/Button/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './Button.scss';
+
+interface ButtonProps {
+  primary?: boolean;
+  children: string;
+  onClick: () => void;
+}
+
+const Button: React.FC<ButtonProps> = (props: ButtonProps) => {
```

Comment:
It's not needed to use ButtonProps several times
Just destructure props in arguments
`const Button: React.FC<ButtonProps> = ({
onClick,
primary,
children,
}) => {`